### DISCUSSION TITLE
Q: 有没有好心人告诉一下Vue2怎么使用waline,搞了半天还是不会 || Q: Can anyone kindly tell me how to use waline in Vue2? I still can’t figure it out after working on it for a long time.

### DISCUSSION THREAD

**Discussion by Apcin** [walinejs/waline#2259] [Q&A] [answered]
### Describe the bug | 问题描述

Leancloud 与vercel均已部署完毕，waline官网上说是基于VUE3，可以给个vue2的demo吗，自己做的网站就差评论区功能没实现了，求帮助。

### Website URL | 问题网站

https://waline.js.org/cookbook/import/project.html#%E5%9C%A8%E6%99%AE%E9%80%9A%E9%A1%B9%E7%9B%AE%E4%B8%AD%E5%AF%BC%E5%85%A5

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Leancloud and vercel have been deployed. The waline official website says it is based on VUE3. Can you give me a demo of vue2? The comment area function of the website I made is not implemented. Please help.

### Website URL | Problem website

https://waline.js.org/cookbook/import/project.html#%E5%9C%A8%E6%99%AE%E9%80%9A%E9%A1%B9%E7%9B%AE%E4 %B8%AD%E5%AF%BC%E5%85%A5

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
vue2 就用类似 HTML 接入的方式就可以了，忘记 vue2, vue3 的存在即可 https://waline.js.org/guide/get-started/#html-%E5%BC%95%E5%85%A5-%E5%AE%A2%E6%88%B7%E7%AB%AF

---
Just use a method similar to HTML access for vue2. Just forget about the existence of vue2 and vue3 https://waline.js.org/guide/get-started/#html-%E5%BC%95%E5%85 %A5-%E5%AE%A2%E6%88%B7%E7%AB%AF
